### DISCUSSION TITLE
What are all of the badges I can get for my profile?

### DISCUSSION THREAD

**Discussion by Fxdli** [community/community#20541] [Other Feature Feedback, Questions, & Ideas] [answered]
What are the achievments I can get?

**Comment by Itz-Hyperz** [comment, marked as answer]
You can get all of the achievements that are publicly listed here with their associated rankings below them.
https://github.com/Schweinepriester/github-profile-achievements

**Reply by Fxdli** [reply]
awesome, thanky ou

**Reply by rajee-a** [reply]
Amazing work 🥳
